src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -97,6 +98,20 @@ public boolean isEnabled(final String toggleName, final UnleashContext context ,
         return Optional.ofNullable(toggleRepository.getToggle(toggleName));
     }
 
+    public List<String> getToggleNames() {
+        return toggleRepository.getToggleNames();
+    }
+
+    @Override
+    public List<String> getEnabledToggleNames() {
+        return getEnabledToggleNames(contextProvider.getContext());
+    }
+
+    @Override
+    public List<String> getEnabledToggleNames(final UnleashContext context) {
+        return toggleRepository.getToggleNames().stream().filter(toggleName -> this.isEnabled(toggleName, context, false)).collect(Collectors.toList());
```

Comment:
For readability this should be split over multiple lines. Generally I would prefer it like this:

```java
return toggleRepository.getToggleNames().stream()
    .filter(toggleName -> this.isEnabled(toggleName, context, false))
    .collect(Collectors.toList());
```